uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -688,34 +712,6 @@ QUERY
 }
 
 
-=pod
-
-=head3 debug_inrange($val, $range)
-
-Will evaluate whether the scalar C<$value> is in the specified C<$range>.
-
-INPUTS:
-  - $val  : scalar value to evaluate
-  - $range: scalar range string
-
-RETURNS: 1 if in range, 0 if not in range
-
-=cut
-
-sub debug_inrange {
```

Comment:
any reason this is being deleted? was it used anywhere?

Reply:
this function was not used anywhere in the code so I deleted it.